docs/content/docs/guide/running.md

```diff
@@ -141,12 +141,16 @@ roses are red, violets are blue. pipeline are bound to flake by design.
 /test <pipelinerun-name>
 ```
 
+For push requests, GitOps commands will be executed on commits like below (only `GitHub` provider is supported)
```

Comment:
I think your comment is too succint and only understand by someone who already understand on how to do this action. can you guide the user to open a commit on Github and click on the line number to add a comment? our users are not necessary advanced github expert

Reply:
Updated ptal